pages/auth/[slug].tsx

```diff
@@ -14,19 +14,26 @@ const Auth = () => {
   }
 
   return (
-    <div className="flex h-screen bg-white">
-      <div className="flex flex-1 flex-col justify-center items-center bg-custom-blue text-white p-8">
-        <div className="mb-8">
-          <Image
-            src="/login-register.svg"
-            alt="Login and Register"
-            width={400}
-            height={60}
-          />
+    <div className="flex flex-col h-screen bg-white">
+      <nav className="custom:flex hidden justify-center items-center bg-[#1C364B] h-16">
+        <div className="flex items-center">
+          <Image src="/logo.svg" alt="Logo" width={120} height={40} />
+        </div>
+      </nav>
+      <div className="flex flex-1">
+        <div className="custom:hidden flex-1 flex flex-col justify-center items-center bg-custom-blue text-white p-8">
+          <div className="flex justify-center items-center">
+            <Image
+              src="/logo.svg"
+              alt="Login and Register"
+              width={400}
+              height={60}
+            />
+          </div>
+        </div>
```

Comment:
Lets extract this into a sub function (can be in the same file)  So is more readable.